apps/backoffice/app/routes/api/telegram.ts

```
@@ -1,37 +1,46 @@
   request: Request,
   context: Route.LoaderArgs["context"],
-  orgId: string | undefined,
+  scopeSegment: string | undefined,
 ) => {
-  if (!orgId) {
-    return new Response("Missing organisation id", { status: 400 });
+  if (!scopeSegment) {
+    return new Response("Missing scope", { status: 400 });
+  }
+
+  let scope;
```

Biome (2.5.0)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)